WalletWasabi.Packager/Program.cs

```diff
@@ -168,30 +168,35 @@ private static async Task PublishAsync()
 		if (Directory.Exists(BinDistDirectory))
 		{
 			await IoHelpers.TryDeleteDirectoryAsync(BinDistDirectory).ConfigureAwait(false);
-			Console.WriteLine($"Deleted {BinDistDirectory}");
+			Console.WriteLine($"# Deleted {BinDistDirectory}");
 		}
 
+		Console.WriteLine($"# Run dotnet restore");
 		StartProcessAndWaitForExit("dotnet", DesktopProjectDirectory, arguments: "restore --locked-mode");
+
+		Console.WriteLine($"# Run dotnet clean");
 		StartProcessAndWaitForExit("dotnet", DesktopProjectDirectory, arguments: "clean --configuration Release");
 
-		var desktopBinReleaseDirectory = Path.GetFullPath(Path.Combine(DesktopProjectDirectory, "bin", "Release"));
-		var libraryBinReleaseDirectory = Path.GetFullPath(Path.Combine(LibraryProjectDirectory, "bin", "Release"));
+		string desktopBinReleaseDirectory = Path.GetFullPath(Path.Combine(DesktopProjectDirectory, "bin", "Release"));
+		string libraryBinReleaseDirectory = Path.GetFullPath(Path.Combine(LibraryProjectDirectory, "bin", "Release"));
+
 		if (Directory.Exists(desktopBinReleaseDirectory))
 		{
 			await IoHelpers.TryDeleteDirectoryAsync(desktopBinReleaseDirectory).ConfigureAwait(false);
-			Console.WriteLine($"Deleted {desktopBinReleaseDirectory}");
+			Console.WriteLine($"#Deleted {desktopBinReleaseDirectory}");
```

Comment:
```suggestion
			Console.WriteLine($"# Deleted {desktopBinReleaseDirectory}");
```
If we want to go with this.